sockeye/inference.py

```diff
@@ -452,31 +460,40 @@ class ModelState:
     A ModelState encapsulates information about the decoder states of an InferenceModel.
     """
 
-    def __init__(self, states: List[mx.nd.NDArray]) -> None:
+    def __init__(self, states: List[mx.nd.NDArray]=None) -> None:
         self.states = states
 
-    def sort_state(self, best_hyp_indices: mx.nd.NDArray):
+    def sort_state(self, best_hyp_indices: mx.nd.NDArray) -> None:
         """
         Sorts states according to k-best order from last step in beam search.
         """
         self.states = [mx.nd.take(ds, best_hyp_indices) for ds in self.states]
 
+    def append_state(self, states: List[mx.nd.NDArray]) -> None:
+        """
+        Stacks states together (representing several sentences)
+        """
+        if self.states:
+            utils.check_condition(len(states) == len(self.states), "State lists have different lengths")
+            self.states = [mx.nd.concat(old_s, new_s, dim=0) for old_s, new_s in zip(self.states, states)]
+        else:
+            self.states = states
 
 class LengthPenalty:
     """
     Calculates the length penalty as:
-    (beta + len(Y))**alpha / (beta + 1)**alpha
+    (gamma + len(Y))**alpha / (gamma + 1)**alpha
```

Comment:
if you rename this parameter, you should also rename the CLI argument, effectively making this a breaking change so the version should be bumped. Do you have a good reason to change it ?

Reply:
Will do, but this is equally confusing to call in beta, while the beta in the paper is something else completely.